packages/js/src/settings/helpers/search.js

```diff
@@ -448,6 +448,16 @@ export const createSearchIndex = ( postTypes, taxonomies, { userLocale } = {} )
 				"gsc",
 			],
 		},
+		ahrefsverify: {
+			route: "/site-connections",
+			routeLabel: __( "Site connections", "wordpress-seo" ),
+			fieldId: "input-wpseo-ahrefsverify",
+			fieldLabel: __( "Ahrefs", "wordpress-seo" ),
+			keywords: [
+				__( "Webmaster", "wordpress-seo" ),
+				__( "Ahrefs", "wordpress-seo" ),
```

Comment:
I think adding the name of the setting is not necessary:
```suggestion
```

Reply:
I have changed everything except the lowercase name. I think that was a mistake from Manuel in the comment since everything is capitalized and Ahrefs is a name so it would make no sense to not do it.